ui/src/components/AppSwitchRoutes.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {useCallback, useEffect, useState} from "react";
+import {Route, Routes, useNavigate} from "react-router-dom";
+import {RouterRoutes} from "../helpers/routes";
+import {useRecoilState} from "recoil";
+import entPageAtom, {Page} from "../recoil/entPage";
+import {toast} from "react-toastify";
+import AuthPageBase from "./Pages/AuthPage/AuthPageBase";
+import InstallPage from "./Pages/AuthPage/InstallPage";
+import LoginPage from "./Pages/AuthPage/LoginPage";
+import LoadingOverlay from "./LoadingOverlay";
+import SystemViewer from "./Pages/SystemViewer/SystemViewer";
+import Api from "../helpers/api";
+import {TrafficPage} from "./Pages/TrafficPage/TrafficPage";
+
+const api = Api.getInstance();
+
+const AppSwitchRoutes = () => {
+
+    const navigate = useNavigate();
+
+    const [isLoading, setIsLoading] = useState(true);
+    const [entPage, setEntPage] = useRecoilState(entPageAtom);
+    const [isFirstLogin, setIsFirstLogin] = useState(false);
+
+    const determinePage =  useCallback(async () => { // TODO: move to state management
```

Comment:
can the `TODO: move to state management` be removed now?

Reply:
sure, removing